lib/ansible/utils/collection_loader/_collection_finder.py

```diff
@@ -41,6 +41,17 @@
     # deprecated: description='TraversableResources fallback' python_version='3.8'
     TraversableResources = object  # type: ignore[assignment,misc]
 
+
+if sys.version_info >= (3, 10):
+    from importlib.resources import files
+else:
+    # deprecated: description='reliable importlib.resources.files' python_version='3.10'
+    def files(name):
+        if (spec := find_spec(name)) is None:
+            raise ImportError(name)
+        origin = pathlib.Path(spec.origin)
+        return origin.parent
```

Comment:
This compat function is needed for now, since we support this for module side tests. I didn't want to do a dep on `importlib_resources` for this like we did before, and this meets the need of the code within.